bindata/network/ovn-kubernetes/pre-puller.yaml

```diff
@@ -0,0 +1,44 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-upgrades-prepuller
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the pre-puller component during upgrades that pulls the image onto the node.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: ovnkube-upgrades-prepuller
+  template:
+    metadata:
+      labels:
+        app: ovnkube-upgrades-prepuller
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: ovn-kubernetes-node
+      hostNetwork: true
+      hostPID: true
```

Comment:
This doesn't need to be hostPID

Reply:
oh yea true, why do we need access to the process namespace of the host? removing this...